awx/main/utils/licensing.py

```diff
@@ -383,12 +384,24 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        license_date = int(attrs.get('license_date', 0) or 0)
+        if HostMetric:
+            automated_instances = HostMetric.objects.count()
+            first_host = HostMetric.objects.order_by('first_automation').first()
+            if first_host:
+                automated_since = int(first_host.first_automation.timestamp())
+            else:
+                automated_since = license_date
+        else:
+            automated_instances = 0
+            automated_since = now()
```

Comment:
I think this else will go away. If it stays though, `automated_since` is a mix of date and seconds since epoch i.e. 

```>>> str(datetime.datetime.now())
'2021-06-07 13:26:27.456871'
```

Reply:
Yea, this needs fixed, incoming.